lessons/Data-Science/ai-decision-trees/langs/en-us/pages/build-and-test.adoc

```diff
@@ -0,0 +1,80 @@
+= Building and Testing a Decision Tree
+
+On @printable-exercise{level-1.adoc}, we decided that the decision mode at the end of the "teens" branch should be "interest in game". Let's create the rest of the tree and test it out. You can refer to the training dataset on @handout{decision-tree-data.adoc}.
+
+== Build and Understand the Tree
+
+@vspace{1ex}
+
+Fill in the remainder of the tree using the algorithm you have learned (left) before filling in the blanks (right).
+
+@vspace{1ex}
+
+[cols="1,1", stripes="none"]
+|===
+
+|
+@ifnotsoln{@center{@image{../images/incomplete-tree.png, 350}}}
+
+@ifsoln{@center{@image{../images/final-tree.png, 350}}}
+
+|
+
+@n The root node of this tree is @fitb{10em}{@ifsoln{age}}.
+
+
+@n The first set of branches includes @fitb{10em}{@ifsoln{teens}}, @fitb{10em}{@ifsoln{twenties}}, and @fitb{10em}{@ifsoln{thirties}}.
+
+@vspace{1ex}
+
+@n Write the rules that this decision tree follows: @ifsoln{Answers will vary.}
+
+
+a) Predict that @fitb{10em}{@ifsoln{interested teens}} will buy the game.
```

Comment:
This comment is relevant here, but also for a number of other fitbrubies where it's a "will" or "will not" answer:

Why not just phrase this as a question, and let kids write yes or no?

Reply:
Because I want them to write the rules that the decision tree follows. 

you disagree with this? Trees are considered simple and flexible because they produce these easy-to-interpret rules.